### PR TITLE
[FIX] slash-menu-height-fix

### LINKED ISSUE #8326: Slash Menu has more height than it's content
## Bug Description

The slash menu takes up more space vertically than needed. 

image

## Expected Behavior

The height should not exceed more than required to wrap element inside it. 
image

### PR DESCRIPTION
**Pull request by siv-io**
FIX #8326 

I've used the `height: fit-content` property just for customSlashMenu but I think it will work for all dropdown menu's. 
I tested it for a few and works fine, but not sure for edge cases. 

Let me know if the height should be changed to `fir-content`

image

### CODE REVIEW

File: packages/twenty-front/src/modules/ui/layout/dropdown/components/DropdownMenu.tsx [hunk 1]
```diff
@@ -18,12 +19,12 @@ const StyledDropdownMenu = styled.div<{
...
     disableBorder ? 'none' : `1px solid ${theme.border.color.medium}`};
   border-radius: ${({ theme }) => theme.border.radius.md};
-
   box-shadow: ${({ theme }) => theme.boxShadow.strong};
 
   display: flex;
 
-  height: 100%;
+  height: ${({ isCustomSlashMenu }) =>
+    isCustomSlashMenu ? 'fit-content' : '100%'};
```
Comment (Bonapara) [inline packages/twenty-front/src/modules/ui/layout/dropdown/components/DropdownMenu.tsx:27]: @Hitarthsheth07 fit-content was not working everywhere?

Comment (Hitarthsheth07) [reply packages/twenty-front/src/modules/ui/layout/dropdown/components/DropdownMenu.tsx:27]: It does work every, should I change it to fit-content

Comment (Bonapara) [reply packages/twenty-front/src/modules/ui/layout/dropdown/components/DropdownMenu.tsx:27]: If it works everywhere yes

Comment (Hitarthsheth07) [reply packages/twenty-front/src/modules/ui/layout/dropdown/components/DropdownMenu.tsx:27]: Ob it!

Comment (Hitarthsheth07) [reply packages/twenty-front/src/modules/ui/layout/dropdown/components/DropdownMenu.tsx:27]: Done!

**Review by Bonapara — APPROVED** [review]

**Pull request merged**